config/default.schema.json

```diff
@@ -45,13 +45,13 @@
 			"type": "string",
 			"description": "global announcement, shown as notification in header on startpage when not loggedin."
 		},
-		"GLOBAL_ANNOUNCEMENT_TEACHER": {
+		"GLOBAL_ANNOUNCEMENT_TEXT": {
 			"type": "string",
-			"description": "global announcement, shown only to teacher"
+			"description": "global announcement, shown to all user roles specified in GLOBAL_ANNOUNCEMENT_ROLES"
 		},
-		"GLOBAL_ANNOUNCEMENT": {
+		"GLOBAL_ANNOUNCEMENT_ROLES": {
 			"type": "string",
-			"description": "global announcement, shown as notification in header on startpage"
+			"description": "roles the text in GLOBAL_ANNOUNCEMENT_TEXT should be shown to"
```

Comment:
comma separated list of roles for which GLOBAL_ANNOUNCEMENT_TEXT is shown to

Reply:
will change that 👍